### ISSUE TITLE
Icon support in button

### ISSUE THREAD

**Issue by alriksson** [soflyy/oxygen-bugs-and-features#773]
**Describe the feature you'd like to see included in Oxygen.**
Add support for icons in buttons

**Examples of this feature or functionality.**
https://docs.elementor.com/article/59-button
https://kb.wpbeaverbuilder.com/article/54-button-module-overview

**Comment by shinoamakusa** [comment]
Themeco Pro has that too ... would like to see icon support in buttons and other elements too and icon can be icon or image ;)

**Comment by swinggraphics** [comment]
This is a basic page builder feature, and I think everyone expects it. I know I did. Watched the Icon Button in 2 Minutes video and thought, "Are you kidding?"

**Comment by jamescrabb** [comment]
You can just use the link wrapper element and style it in to a button the. You can have whatever you want in the button.

Button is just the appearance of a link.

**Comment by swinggraphics** [comment]
> You can just use the link wrapper element and style it in to a button

Yes, that is the problem. You have to do it all manually. Try explaining the process to a client who just wants to add a simple button with a cute little icon in it. And if you already created a button that you later want to add an icon to, well that's a total hassle. Perhaps the solution is for the Button component just to be a glorified link so that we can easily do whatever we want with it.
